docs/src/test/scala/docs/http/scaladsl/server/directives/JsonStreamingExamplesSpec.scala

```diff
@@ -18,31 +18,29 @@ import scala.concurrent.Future
 
 class JsonStreamingExamplesSpec extends RoutingSpec {
 
-  //#models
+  //#tweet-model
   case class Tweet(uid: Int, txt: String)
-  case class Measurement(id: String, value: Int)
-  //#models
+  //#tweet-model
 
   val tweets = List(
     Tweet(1, "#Akka rocks!"),
     Tweet(2, "Streaming is so hot right now!"),
     Tweet(3, "You cannot enter the same river twice."))
   def getTweets = Source(tweets)
 
-  //#formats
-  object MyJsonProtocol
+  //#tweet-format
+  object MyTweetJsonProtocol
     extends akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
     with spray.json.DefaultJsonProtocol {
 
     implicit val tweetFormat = jsonFormat2(Tweet.apply)
-    implicit val measurementFormat = jsonFormat2(Measurement.apply)
```

Comment:
Same for the case class definition, moving `measurementFormat` to a separate place for the measurement section.